test/e2e/searchResults.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { test, expect } from "@playwright/test";
+
+import { screenshot } from "../utils/screenshot";
+
+test.describe("Search results page", () => {
+	test.beforeEach(async ({ page }) => {
+		await page.goto("/equilibrium/search?query=Oracle.NewPrice");
+		await page.waitForTimeout(1000); // TODO find out better waiting until page is loaded
+	});
+
+	test("shows search results page", async ({ page }) => {
+		await page.waitForTimeout(1000); // TODO find out better waiting until page is loaded
```

Comment:
tohle cekani je to proc? nestacilo se to nacist? zvys spis ten timeout v `beforeEach`